app/classifier/tasks/survey/index.cjsx

```diff
@@ -63,11 +64,20 @@ module.exports = React.createClass
     newAnnotation = Object.assign {}, @props.annotation, _choiceInProgress: true
     @props.onChange newAnnotation
 
+  handleRemove: (choiceID) ->
+    selectedChoices = (item.choice for item in @props.annotation.value)
+    index = selectedChoices.indexOf choiceID
```

Comment:
I think this is a bug in the survey task. Clicking a selected choice should probably edit the annotation, not create a new one.